react/LocationInput.tsx

```diff
@@ -39,14 +42,39 @@ const LocationInput: React.FC<Props> = ({
 
   const countryRules = country ? rules[country] : undefined
 
+  const formatAndValidate = () => {
+    const postalCodeMask = countryRules?.fields.postalCode?.mask
+
+    if (!postalCodeMask) {
+      return
+    }
+
+    setInputValue(msk(inputValue, postalCodeMask))
+
+    const patternInvalid =
+      inputValue.length &&
+      countryRules?.fields.postalCode?.pattern != null &&
+      inputValue.match(countryRules.fields.postalCode.pattern) === null
+
+    if (patternInvalid) {
+      setInvalidPostalCode(true)
+    }
+  }
+
+  useEffect(() => {
+    setInvalidPostalCode(false)
+    formatAndValidate()
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [countryRules])
+
   useEffect(() => {
     let cancelled = false
 
     if (data?.getAddressFromPostalCode) {
       // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
       ;(onSuccess?.(data.getAddressFromPostalCode) || Promise.resolve())
         .then(() => {
-          setAddress(prevAddress => ({
+          setAddress((prevAddress) => ({
```

Comment:
did you update the lint packages? it seems it isn't updated in the GH actions, maybe some file is being ignored by gitignore?

Reply:
I've updated in another branch, weird. The GH actions should run the same linter as the current repo state, right? instead of the newest VTEX linter version